app/components/download_all_button_component.rb

```diff
@@ -8,10 +8,10 @@ def initialize(document:)
 
   def link_options
     # This onclick handler prevents the accordion which contains this element from folding.
-    options = { onclick: 'event.stopPropagation()' }
+    options = {onclick: "event.stopPropagation()"}
```

Comment:
Omit spaces inside braces for hashes. Meant to make hashes visually distinct from blocks.